testsupport/tier_setup.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
 	"github.com/codeready-toolchain/toolchain-common/pkg/test"
-	. "github.com/codeready-toolchain/toolchain-e2e/testsupport/wait" // nolint: golint
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport/wait" // nolint:revive
```

Comment:
what is ignored here?

Reply:
dot imports are not recommended. I updated the `nolint` comment since the `golint` linter is now deprecated (and disabled) and replaced by `revive`